ConsoleUI/Properties/Resources.fr-FR.resx

```diff
@@ -0,0 +1,387 @@
+  <data name="ModListNewMods" xml:space="preserve"><value>{0} new mods available since last update. Show them?</value></data>
+  <data name="ModListScanBad" xml:space="preserve"><value>{0} The repo has not been saved.</value></data>
+  <data name="ModListExportPrefix" xml:space="preserve"><value>install</value></data>
+  <data name="ModListExported" xml:space="preserve"><value>Mod list exported to {0}</value></data>
+  <data name="ModListExportFailed" xml:space="preserve"><value>Export failed: {0}</value></data>
+  <data name="ModListHelpSymbolHeader" xml:space="preserve"><value>Status Symbols</value></data>
+  <data name="ModListHelpInstalled" xml:space="preserve"><value>Installed</value></data>
+  <data name="ModListHelpAutoInstalled" xml:space="preserve"><value>Auto-installed</value></data>
+  <data name="ModListHelpUpgradeable" xml:space="preserve"><value>Upgradeable</value></data>
+  <data name="ModListHelpManuallyInstalled" xml:space="preserve"><value>Manually installed</value></data>
+  <data name="ModListHelpReplaceable" xml:space="preserve"><value>Replaceable</value></data>
+  <data name="ModListHelpUnavailable" xml:space="preserve"><value>Unavailable</value></data>
+  <data name="ModListHelpBasicKeysHeader" xml:space="preserve"><value>Basic Keys</value></data>
+  <data name="ModListHelpMoveFocus" xml:space="preserve"><value>Move focus</value></data>
+  <data name="ModListHelpSelectRow" xml:space="preserve"><value>Select row</value></data>
+  <data name="ModListHelpClearSearch" xml:space="preserve"><value>Clear search</value></data>
+  <data name="ModListHelpSpecialSearchesHeader" xml:space="preserve"><value>Special Searches</value></data>
+  <data name="ModListHelpAuthor" xml:space="preserve"><value>author</value></data>
+  <data name="ModListHelpName" xml:space="preserve"><value>name</value></data>
+  <data name="ModListHelpSearchAuthor" xml:space="preserve"><value>Mods by author</value></data>
+  <data name="ModListHelpSearchInstalled" xml:space="preserve"><value>Installed mods</value></data>
+  <data name="ModListHelpSearchUpgradeable" xml:space="preserve"><value>Upgradeable mods</value></data>
+  <data name="ModListHelpSearchDepends" xml:space="preserve"><value>Depend on name</value></data>
+  <data name="ModListHelpSearchConflicts" xml:space="preserve"><value>Conflict w/ name</value></data>
+  <data name="ModListHelpSearchNew" xml:space="preserve"><value>New mods</value></data>
+  <data name="ExitTitle" xml:space="preserve"><value>{0}, the Comprehensive Kerbal Archive Network</value></data>
+  <data name="ExitBody" xml:space="preserve"><value>YOU ARE USING {1}.
+
+Thanks for downloading {0}. We hope you have as
+much fun using it as we had (and have) making it.
+
+If you have paid for {0}, try to get your money back,
+because you can download {0} for free from
+{2}
+
+If you have any problems using {0}, please send us an issue at
+{3}
+
+{0} WAS CREATED BY THE {0} AUTHORS:
+{4}
+</value></data>
```

Comment:
Thanks for this!

Just an FYI that these lines are still in English. If you're still planning to look at them, that's fine and there's no hurry. But if you already translated them, then there may have been a mix-up with your files before this was committed.

Reply:
Thanks for the advice!
I made a mistake earlier and committed a partial translation of console UI instead of the complete translation of Core. I did rebase, but I kept looking for a file I had never committed. Took me a while to understand 😅 
I am planning to translate this file, I just prematurely committed it.
Hopefully, I'll be done by the end of the week 🙂